README.md

```diff
@@ -71,4 +71,22 @@ export default Vue.extend({
 	@remove="handleRemoveEvent"
 	@searchChange="handleSearchChangeEvent"
 />
+```
+
+```scss
```

Comment:
I haven't put the full explanation in the README here quite yet. But what I'm anticipating is that, initially, we'll do this all in the context of SCSS (since that's what *we* need first)

We would try to find a good balance of **the most** common properties (esp. color) that people would want to style on the selects. Provide those as named arguments for the SCSS mixins, and then each project should run the mixins w/ `@include` to get their base styles

*Then* we provide some documentation on the most common & stable classes that users might want to target to override specific things. Devs would write those overrides in their own projects, with simple CSS or SCSS under the mixins, to target whatever specific classes they're interested in